docs/reference/commands/certutil.asciidoc

```diff
@@ -176,14 +176,17 @@ with the `ca` parameter.
 
 `--pass <password>`:: Specifies the password for the generated private keys.
 +
-Keys stored in PKCS#12 format are always password protected.
+Keys stored in PKCS#12 format are always password protected. However, it supports
+the password to be _blank_. If you want to specify a _blank_ password without 
+prompt, use '--pass ""' (with no '=').
```

Comment:
I like the way this has been structured, but I think we can tighten up the wording a bit.
What do you think of:
```
Keys stored in PKCS#12 format are always password protected, however,
this password may be _blank_. If you want to specify a blank password
without a prompt, pass `--pass ""` (with no `=`) on the command line.
```